repo-tracker-app/src/App.css

```diff
@@ -0,0 +1,118 @@
+@import url('https://fonts.googleapis.com/css2?family=PT+Serif&display=swap');
+
+body{
+    font-family: 'PT Serif';
+    background-color: white;
+}
+
+h1{
+    font-size: 220%;
+}
+
+h1, h3, h4, p{
+    text-align: center;
+}
+
+img{
+    height:40px;
+    vertical-align: middle;
+    margin-right: 10px;
+}
+
+h1>a, h3>a{
+    text-decoration: none;
+    color: black;
+}
+
+form{
+    text-align: center;
+}
+
+#repos-grid{
+    margin-top: 15px;
+    display: grid;
+    grid-template-columns: 30% 30% 30%;
+    column-gap: 3%;
+    row-gap: 15px;
+}
+
+#repos-grid>div{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-color: white;
+    border: indianred 1px solid;
+    box-shadow: rgb(143, 63, 63) 2px 2px;
+    padding: 15px 0px 15px 0px;
+    font-size: 105%;
+}
+
+#repos-grid>div:hover{
+    cursor: pointer;
+    background-color: rgb(255, 230, 230);
+    box-shadow: rgb(143, 63, 63) 4px 4px;
+}
+
+li{
+    list-style: none;
+}
+
+a{
+    text-decoration: none;
+}
+
+#featured-box{
+    width: 90%;
+    height: 100%;
+    justify-content: space-evenly;
+    border: indianred 1px solid;
+    box-shadow: rgb(143, 63, 63) 2px 2px;
+    margin-top: 15px;
+
+}
+
+#featured-data{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    margin-bottom: 25px;
+}
+
+#featured>ul{
+    margin-right: 3%;
+}
+
+#featured-data>ul>:first-child{
+    font-weight: bold;
+}
+
+#percent-container{
+    display:flex;
+    flex-direction: column;
+}
+
+#percentages{
+    display: flex;
+}
+
+.flex-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.flex-container>a{
+    color: black;
+}
+
+@media screen and (max-width: 907px){
+    #repos-grid{
+        grid-template-columns: 46% 46%;
+    }
+}
+
+@media screen and (max-width: 612px){
+    #repos-grid{
+        grid-template-columns: 92%;
+    }
+}
```

Comment:
Nice styling, but it would be nice if it looked similar to the actual repositories page.